docs/NEMU + XS-GEM5 使用流程.md

```
@@ -0,0 +1,698 @@
+   https://github.com/OpenXiangShan/GEM5
+
+   https://docs.xiangshan.cc/zh-cn/latest/tools/simpoint/
+
+
+## 1.环境安装
+
+​	本教程是在 Ubuntu 22.04 的环境上完成的，接下来将详细说明其余环境的安装过程。
+
+### 1.1 gcc 安装
+
+​	本教程所采用的的 gcc 版本为 15.2.0。
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 2

(MD010, no-hard-tabs)